src/components/v5/common/ActionSidebar/ActionSidebar.tsx

```diff
@@ -20,6 +21,15 @@ import { ActionSidebarProps } from './types';
 
 const displayName = 'v5.common.ActionSidebar';
 
+const variants = {
+  hidden: {
+    x: '100%',
+  },
+  visible: {
+    x: 0,
+  },
+};
+
```

Comment:
maybe we should move this const to some separated file?

Reply:
Fixed